forwards_paper.tex

```diff
@@ -707,12 +730,18 @@ \subsubsection*{Sequential simplification and prior history}
 there is no computational advantage to simplifying more often than is necessary
 to keep memory usage down.
 
+\krt{Empirically, I did find that simplifying more often than every 1,000 generations gave a run-time hit of about 25
+percent???}
+
```

Comment:
oh! right; we should say that "so the most efficient strategy is to simplify only when is necessary to keep memory usage down"

Reply:
This has to do with the memory-vs-speed tradeoff above; we should mention this only in one place (here?).